packages/web_drivers/lib/safari_driver_runner.dart

```diff
@@ -0,0 +1,88 @@
+/// Running Safari Driver which comes installed in every MacOS operation system.
+///
+/// The version of driver is the same as the version of the Safari installed
+/// in the system.
+import 'dart:io' as io;
+
+class SafariDriverRunner {
+  /// Start Safari Driver installed in the MacOS operating system.
+  ///
+  /// If a specific version is requested, it will check the existing system
+  /// version and throw and exception the versions do not match.
+  ///
+  /// If the operating system is not MacOS, the method will throw an exception.
+  Future<void> start({String version}) async {
+    if (!io.Platform.isMacOS) {
+      throw AssertionError('The operating system should be MacOS: '
+          '${io.Platform.operatingSystem}');
+    }
+    final io.Process process = await runDriver(version: version);
+    final int exitCode = await process.exitCode;
+    if (exitCode != 0) {
+      throw Exception('Driver failed with exitcode: $exitCode');
+    }
+  }
+
+  /// Safari Driver needs to be enabled before it is used.
+  ///
+  /// The driver will need to get re-enabled after the machine is restarted.
+  ///
+  /// The enabling waits for user input for authentication.
+  void _enableDriver() {
+    io.Process.runSync('//usr/bin/safaridriver', ['--enable']);
+  }
+
+  /// Compare the version of the installed driver to the requested version.
+  ///
+  /// Throw an exception if they don't match.
+  Future<void> _compareDriverVersion(String version) async {
+    io.Process.run('//usr/bin/safaridriver', ['--version']);
+
+    final io.ProcessResult versionResult =
+        await io.Process.run('//usr/bin/safaridriver', ['--version']);
+
+    if (versionResult.exitCode != 0) {
+      throw Exception('Failed to get the safari driver version.');
+    }
+    // The output generally looks like: Included with Safari 13.0.5 (14608.5.12)
+    final String output = versionResult.stdout as String;
+    final String rest =
+        output.substring(output.indexOf('Safari'));
+
+    print('INFO: driver version in the system: $rest');
+
+    // Version number such as 13.0.5.
+    final String versionAsString = rest.trim().split(' ')[1];
+
+    if (versionAsString != version) {
+      throw Exception('System version $versionAsString not match requested '
+          'version $version');
+    }
+  }
+
+  /// Run Safari Driver installed in the MacOS operating system and return
+  /// the Process.
+  ///
+  /// Returns a [:Future<Process>:] that completes with a
```

Comment:
Is [:___:] valid dartdoc syntax? I've never seen it before and can't find any docs on it. How does it work? (maybe I should use it in my docs too)